sceptre/securitycentral/templates/sechub_suppress_infra.yaml

```diff
@@ -0,0 +1,273 @@
+      VisibilityTimeout: 130
+    UpdateReplacePolicy: Delete
+    DeletionPolicy: Delete
+  SecurityHubFindingsDeadLetterQueuePolicy:
+    Type: AWS::SQS::QueuePolicy
+    Properties:
+      PolicyDocument:
+        Statement:
+        - Action: sqs:*
+          Condition:
+            Bool:
+              aws:secureTransport: 'false'
+          Effect: Deny
+          Principal: "*"
+          Resource:
+            Fn::GetAtt:
+            - SecurityhubFindingsDeadLetterQueue
+            - Arn
+          Sid: Enforce TLS for all principals
+        Version: '2012-10-17'
+      Queues:
+      - Ref: SecurityhubFindingsDeadLetterQueue
+  SecurityHubFindingsQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      KmsMasterKeyId:
+        Fn::GetAtt:
+        - SqsEncryptionKey
+        - Arn
+      RedrivePolicy:
+        deadLetterTargetArn:
+          Fn::GetAtt:
+          - SecurityhubFindingsDeadLetterQueue
+          - Arn
+        maxReceiveCount: 1
+      VisibilityTimeout: 130
+    UpdateReplacePolicy: Delete
+    DeletionPolicy: Delete
+  SecurityHubFindingsQueuePolicy:
+    Type: AWS::SQS::QueuePolicy
+    Properties:
+      PolicyDocument:
+        Statement:
+        - Action: sqs:*
+          Condition:
+            Bool:
+              aws:secureTransport: 'false'
+          Effect: Deny
+          Principal: "*"
+          Resource:
+            Fn::GetAtt:
+            - SecurityHubFindingsQueue
+            - Arn
+          Sid: Enforce TLS for all principals
+        - Action:
+          - sqs:SendMessage
+          - sqs:GetQueueAttributes
+          - sqs:GetQueueUrl
+          Effect: Allow
+          Principal:
+            Service: events.amazonaws.com
+          Resource:
+            Fn::GetAtt:
+            - SecurityHubFindingsQueue
+            - Arn
+        Version: '2012-10-17'
+      Queues:
+      - Ref: SecurityHubFindingsQueue
+
+  # In every account, suppress findings for CIS4.3 and CIS1.14
+  SecurityHubSuppressionruleCIS43:
+    Type: AWS::Events::Rule
+    Properties:
+      Description: SecHubSuppress CIS4.3 and 1.14 for all accounts
+      EventPattern:
+        detail:
+          findings:
+            GeneratorId:
+              - 'arn:aws:securityhub:::ruleset/cis-aws-foundations-benchmark/v/1.2.0/rule/4.3'
+              - 'arn:aws:securityhub:::ruleset/cis-aws-foundations-benchmark/v/1.2.0/rule/1.14'
+            Workflow:
+              Status:
+              - NEW
+        detail-type:
+        - Security Hub Findings - Imported
+        source:
+        - aws.securityhub
+      State: ENABLED
+      Targets:
+      - Arn:
+          Fn::GetAtt:
+          - SecurityHubFindingsQueue
+          - Arn
+        Id: Target0
+
+Outputs:
+  SecurityHubBatchUpdateSuppressionRoleArn:
+    Value: !GetAtt SecurityHubBatchUpdateSuppressionRole.Arn
+    Description: Arn of batch update role
+    Export:
+      Name: !Sub '${AWS::Region}-${AWS::StackName}-SecurityHubBatchUpdateSuppressionRoleArn'
```

Comment:
we don't need `${AWS::Region}` in outputs because outputs are unique per region.